src/bikeracks/__init__.py

```diff
@@ -42,21 +49,36 @@ def create_app(test_config=None):
         pass
         
     # register close_db and init_db_command with the app
-    from . import db
     db.init_app(app)
     
     # import and register the blueprint from the factory
-    from . import bikes
-    from . import votes
     app.register_blueprint(bikes.bikes)
     app.register_blueprint(votes.votes)
         
     @app.route('/')
     def home(name=None):
         # update map with approved markers
-        # render_template invokes the jinja template engine
         return render_template('base.html', name=name)
         
+    @app.route('/submitFeedback', methods=('POST',))
+    def submitFeedback():
+        csv_file = os.path.join(app.instance_path, "feedback.csv")
+        timestamp = int(time())
+        feedback = request.form.get('feedback', type=str)
+        if len(feedback) > 280:
```

Comment:
Much better.

Probably you are better off with a `validateFeedback(feedback)` function or something that checks everything you might care about.

Reply:
What else would I check?

Reply:
Looks like Flask already parses the incoming bytes as 'utf-8' and replaces invalid characters, so you don't have to worry about that.

This is probably fine. If you get feedback that you think is garbage then you can think about how to better lock this down and prevent it later.